site/content/docs/include/storage.en.md

```diff
@@ -38,13 +38,13 @@ volumes:
 <span class="parent-field">storage.volumes.</span><a id="volume" href="#volume" class="field">`volume`</a> <span class="type">Map</span>  
 Specify the configuration of a volume.
 
-<span class="parent-field">volume.</span><a id="path" href="#path" class="field">`path`</a> <span class="type">String</span>  
+<span class="parent-field">storage.volumes.volume.</span><a id="path" href="#path" class="field">`path`</a> <span class="type">String</span>  
```

Comment:
Can we make this field consistent with [this](https://aws.github.io/copilot-cli/docs/manifest/pipeline/#stages-deployments-name), which seems to make more sense.